uqcsbot/scripts/events.py

```diff
@@ -0,0 +1,120 @@
+from typing import List
+from uqcsbot import bot, Command
+from icalendar import Calendar, vText
+from requests import get
```

Comment:
importing `requests` plain and using `requests.get` is more common - `get` as a plain verb is ambiguous when reading later code.

It might be good to also look into us doing HTTP sessions properly, not in the scope of this PR though